tendrl/common/log.py

```diff
@@ -20,9 +16,3 @@ def setup_logging(
         logging.config.dictConfig(log_config)
     else:
         raise Exception("logging configuration not found")
```

Comment:
I understand but there is no generic exception defined for this scenario. We would need one for this. Can be created in this source only may be.